Tetris (.NET Core)/TetrisConsoleWriter.cs

```diff
@@ -31,7 +31,7 @@ public class TetrisConsoleWriter
             Console.WindowWidth = this.consoleColumns;
             Console.BufferHeight = this.consoleRows + 1;
             Console.BufferWidth = this.consoleColumns;
-            Console.ForegroundColor = ConsoleColor.Black;
+            Console.ForegroundColor = ConsoleColor.White;
```

Comment:
Don't commit other changes.

Reply:
Done. I used the white color because its better seen on my screen.